src/types/types.ts

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from 'react';
+import styles from '../components/Carousels.module.css';
+import { ResponsiveSettings } from '../interfaces/interfaces';
+
+export type TextLink = {
+  text: string;
+  link: string;
+};
+
+export type ChildCard = {
```

Comment:
if type related only for one component, then you should describe it in the component file.